tools/compile_tf_graph.py

```diff
@@ -143,13 +170,37 @@ def post_compile(cls, rec_layer_name, network, output_file_name=None):
     assert rec_layer_name in network.layers
     rec_layer = network.layers[rec_layer_name]
     assert isinstance(rec_layer, RecStepByStepLayer)
-    info = {"state_vars": {}, "stochastic_var_order": [], "stochastic_vars": {}}
-    init_ops = []
-    tile_batch_repetitions = tf.placeholder(name="tile_batch_repetitions", shape=(), dtype=tf.int32)
-    tile_batch_ops = []
-    src_beams = tf.placeholder(name="src_beams", shape=(None, None), dtype=tf.int32)  # (batch,beam)
-    select_src_beams_ops = []
-    next_step_ops = []
+    cell = rec_layer.cell
+    assert isinstance(cell, SubnetworkRecCellSingleStep)
+
+    # tmp structures
+    init_base_ops_list = []
+    init_ops_list = []
+    update_ops_list = []
+    post_update_ops_list = []
+
+    # specific collections needed by sprint #
+    update_ops_coll = tf.get_collection_ref("update_ops")
+    post_update_ops_coll = tf.get_collection_ref("post_update_ops")
+    encode_ops_coll = tf.get_collection_ref("encode_ops")
+    decode_ops_coll = tf.get_collection_ref("decode_ops")
+    decoder_input_vars_coll = tf.get_collection_ref("decoder_input_vars")
+    decoder_output_vars_coll = tf.get_collection_ref("decoder_output_vars")
+    global_vars_coll = tf.get_collection_ref("global_vars")
+
+    # json info #
```

Comment:
Do not make comments like this. Why the ending `#`? Make it consistent to like all the other comments in the code.

Reply:
Is this fixed now? GitHub says outdated. Please mark as resolved when you fixed sth.